### PR TITLE
AMO 1503 report

### PR DESCRIPTION
**Pull request by luohezhiming**
## Fixes/Resolves:

(replace this with the issue # fixed or resolved, if no issue exists then a brief statement of what this PR does)

## Summary/Motivation:
Modify report for AMO 1503

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by tarnold17 — APPROVED** [review]

File: watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/swine_wwt/swine_wwt.py [hunk 1]
```diff
@@ -639,72 +688,371 @@ def total_operating_cost(b):
...
+    print(f"Water recovery: {volumetric_recovery:.3f} m3 of product/m3 of feed")
+    CODR_normalized = value(
+        pyunits.convert(
+            1
+            - m.fs.product_water.properties[0].flow_mass_comp["cod"]
+            / m.fs.feed.properties[0].flow_mass_comp["cod"],
+            to_units=pyunits.dimensionless,
         )
+    )
+    print(f"COD removal: {CODR_normalized:.4f} dimensionless")
```
Comment (adam-a-a) [inline watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/swine_wwt/swine_wwt.py]:
```suggestion
    print(f"COD removal: {CODR_normalized:.4f} kg COD removed/ kg COD in")
```

File: watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/swine_wwt/swine_wwt.py [hunk 2]
```diff
@@ -639,72 +688,371 @@ def total_operating_cost(b):
...
+    print(f"COD removal: {CODR_normalized:.4f} kg COD removed/ kg COD in")
+    TPR_normalized = value(
+        pyunits.convert(
+            1
+            - m.fs.product_water.properties[0].flow_mass_comp["phosphates"]
+            / m.fs.feed.properties[0].flow_mass_comp["phosphates"],
+            to_units=pyunits.dimensionless,
+        )
+    )
+    print(f"Total P removal: {TPR_normalized:.4f} dimensionless")
```
Comment (adam-a-a) [inline watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/swine_wwt/swine_wwt.py]:
```suggestion
    print(f"Total P removal: {TPR_normalized:.4f} kg P removed/ kg P in")
```

File: watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/swine_wwt/swine_wwt.py [hunk 3]
```diff
@@ -639,72 +688,371 @@ def total_operating_cost(b):
...
+    TNR_normalized = value(
+        pyunits.convert(
+            1
+            - (m.fs.product_water.properties[0].flow_mass_comp["ammonium_as_nitrogen"])
+            / (m.fs.feed.properties[0].flow_mass_comp["ammonium_as_nitrogen"]),
+            to_units=pyunits.dimensionless,
+        )
+    )
+    print(f"Total N removal: {TNR_normalized:.4f} dimensionless")
```
Comment (adam-a-a) [inline watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/swine_wwt/swine_wwt.py]:
```suggestion
    print(f"Total N removal: {TNR_normalized:.4f} kg N removed/ kg N in")
```

**Review by adam-a-a — APPROVED** [review]
Great job!

**Pull request merged**